src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonUtilities.Schema.cs

```diff
@@ -176,6 +180,11 @@ private static JsonElement GetJsonSchemaCached(JsonSerializerOptions options, Fu
 #endif
     }
 
+#if !NET9_0_OR_GREATER
```

Comment:
NB I made the decision to suppress trimmer warnings for this API to minimize friction when consuming the library in older TFMs.